frontend/src/components/pod/List.tsx

```diff
@@ -207,13 +216,15 @@ export function PodListRenderer(props: PodListProps) {
       ]}
       data={pods}
       reflectInURL={reflectTableInURL}
+      clusterErrors={clusterErrors}
```

Comment:
in what PR `clusterErrors` prop is implemented for ResourceListView?

Reply:
It was implemented for ResourceTable in this PR (ResourceListView extends ResourceTable so gets the prop from there): 
- https://github.com/headlamp-k8s/headlamp/pull/2436